doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>Typos:</p><ul><li>To demonstrate <strong>the</strong> filling of a masked word task<strong>,</strong> we will use a <a href="https://huggingface.co/StanfordAIMI/RadBERT" target="_blank">RadBERT model</a> which was pre-trained on radiology reports.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>